bin/main.rb

```diff
@@ -1,2 +1,71 @@
 #!/usr/bin/env ruby
-puts "Hello World"
\ No newline at end of file
+# this method creates the board for the game
+def create_board
+  matrix = Array.new(9)
+  i = 0
+  while i < matrix.length
+    matrix[i] = '-'
+    i += 1
+  end
+  matrix
+end
+
+# this method checks if the selected cell is empty: Empty means containing a '-'
+def check_matrix_cell(cell_to_check)
```

Comment:
[OPTIONAL] A method like this would be a good candidate for Board class in next milestone

Reply:
Thank you, we'll keep that in mind :)